spring-cloud-gateway-core/src/main/java/org/springframework/cloud/gateway/config/HttpClientProperties.java

```diff
@@ -293,6 +296,64 @@ public String toString() {
 		/** The default ssl configuration type. Defaults to TCP. */
 		private SslProvider.DefaultConfigurationType defaultConfigurationType = SslProvider.DefaultConfigurationType.TCP;
 
+		/** Spring Cloud Gateway keystore path. */
```

Comment:
Please remove the "Spring Cloud Gateway" prefix. As part of the generated docs it will include "spring.cloud.gateway.httpclient". Same applies to all new fields.

Reply:
incorporated in the latest commit